src/components/field/EncryptDB.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react'
+import PropTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { CheckboxField } from './CheckboxField'
+import DialogEncrypt from '../dialog/dialog-encrypt'
+
+const CODE = 'encryptDB'
+const LABEL = i18n.t('Encrypt device database')
+const HELPTEXT = i18n.t(
+    'Encrypt all data stored on device. Data can be lost if there are problems with an encrypted database. This will not affect the DHIS2 database stored on an external server.'
```

Comment:
I'm not sure if this is still the case, but we've had problems with strings not translating when they are defined outside of a function (@amcgee has this been solved? - I'm thinking yes, but want to check)

Reply:
This should be OK in platform applications which fetch and load the relevant locale before initializing the application code, but it is safer to ensure that `i18n.t` is always called at render time, rather than at module load time.  This way, if the React tree is re-rendered after switching locales it will correctly pick up the new language.